cypress/utils/utils.ts

```diff
@@ -35,3 +36,12 @@ export function selectItemsPerPage(items: number): void {
     cy.get(commonView.itemsPerPageMenuOptions);
     cy.get(`li > button[data-action="per-page-${items}"]`).click();
 }
+
+export function selectFormItems(fieldId: string, item: string): void {
+    cy.get(fieldId).click();
+    cy.contains("button", item).click();
+}
+
+export function checkFlashMessage(fieldId: string, message: string): void {
```

Comment:
This can be more specific, as I see that notifications are generated only on creating new control members. Message string is for success in all cases. 
The function name can be something like getSuccessAlert() or getAlertNotification() ... etc with message string almost fixed, so that instead of message param, we can pass the entity name, which was created.

Reply:
I agree with you. Thanks for suggestion.
 @nitishSr I will update.

Reply:
We can keep the string in that function(getSuccessAlert) because the alert differs for different entities.
for ex. 
1] stakeholder group :- Success! **${stkaholdergroupName}** was added as a **stakeholder group**.
2] stakeholder :- Success! **${stakeholderName}** was added as a **stakeholder**.